docs/core/porting/third-party-deps.md

```diff
@@ -1,38 +1,40 @@
 ---
-title: Porting to .NET Core - Analyzing your Third-Party Party Dependencies
-description: Porting to .NET Core - Analyzing your Third-Party Dependencies
-keywords: .NET, .NET Core
+title: Porting to .NET Core - Analyze your third-party dependencies
+description: Learn how to analyze third-party dependencies in order to port your project from .NET Framework to .NET Core.
 author: cartermp
 ms.author: mairaw
-ms.date: 06/20/2016
+ms.date: 02/01/2018
 ms.topic: article
 ms.prod: .net-core
-ms.devlang: dotnet
 ms.assetid: b446e9e0-72f6-48f6-92c6-70ad0ce3f86a
 ---
+# Analyze your third-party dependencies
 
-# Porting to .NET Core - Analyzing your Third-Party Party Dependencies
+If you're looking to port your code to .NET Core or .NET Standard, the first step in the porting process is to understand your third-party dependencies. Third-party dependencies are either [NuGet packages](#analyze-referenced-nuget-packages-on-your-project) or [DLLs](#analyze-dependencies-that-arent-nuget-packages) you're referencing on your project. Evaluate each dependency and develop a contingency plan for the dependencies that aren't compatible with .NET Core. This topic shows you how to determine if the dependency is compatible with .NET Core.
 
-The first step in the porting process is to understand your third party dependencies.  You need to figure out which of them, if any, don't yet run on .NET Core, and develop a contingency plan for those which don't run on .NET Core.
+## Analyze referenced NuGet packages in your project
 
-## Prerequisites
+If you're referencing NuGet packages in your project, you need to verify if they're compatible with .NET Core.
+There are a few ways to accomplish that:
 
-This article will assume you are using Windows and Visual Studio, and that you have code which runs on the .NET Framework today.
+* Using the NuGet Package Explorer app (most reliable method).
```

Comment:
These could be links to the relevant sections. I think that that would enhance the readability of the topic.
also, most --> the most